lib/meadow/data/file_set.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Meadow.Data.FileSet do
+  @moduledoc """
+  FileSets are used to describe objects stored in Amazon S3
+  """
+
+  import Ecto.Changeset
+  use Ecto.Schema
+
+  embedded_schema do
```

Comment:
We need accession_number (unique) and description and role (am/pm). (these come in on the Ingest Sheet)

Do we need a table for FileSet ?